docs/telemetry.md

```diff
@@ -22,32 +92,149 @@ An example of the data collected is the following:
 {"key" : "hardware_arch","value" : "x86_64 x86_64"}]}]
 ```
 
+## Telemetry Agent
+
+The Percona Telemetry agent functions as a dedicated OS daemon process. The service name on the operating system is `percona-telemetry-agent`. The agent can create, read, write, and delete files in the `${telemetry root path}` and only interacts with JSON files.
```

Comment:
Please add that agent log file can be found at `/var/log/percona/telemetry-agent.log` (this is for both rpm and deb)